### DISCUSSION TITLE
New Emotes

### DISCUSSION THREAD

**Discussion by igorchook** [f3l1x98/blish-emotes#72] [Ideas]
Hello!
Are there any plans to update this with all the new emotes? 
It's been almost a year since the last update.
Pleeaaaaase :)

**Comment by f3l1x98** [comment]
Yes there is.
Unfortunately I have been a little bit busy and to be honest forgot to update.
I will see that I will get it updated as fast as possible, though it might take a few days.

**Reply by igorchook** [reply]
That's fantastic news! Thanks so much 😊

**Reply by f3l1x98** [reply]
I will update tommorrow (I am finished, just need to to a few more release management things and I do not have the time today as Wing 8 takes precedence sorry 😄 )

**Reply by f3l1x98** [reply]
Release is out ([v1.3.1](https://github.com/f3l1x98/blish-emotes/releases/tag/v1.3.1)) so I will close this 😄
